pkg/cluster/ports/ports.go

```diff
@@ -31,9 +31,6 @@ const (
 	InsecureKubeControllerManagerPort = 10252
 	// KubeletReadOnlyPort exposes basic read-only services from the kubelet.
 	// May be overridden by a flag at startup.
-	// This is necessary for heapster to collect monitoring stats from the kubelet
-	// until heapster can transition to using the SSL endpoint.
-	// TODO(roberthbailey): Remove this once we have a better solution for heapster.
 	KubeletReadOnlyPort = 10255
```

Comment:
The comment was removed but not the port constant?

Reply:
The port is still used in kubemark.

We may remove this port once read-only port is not supported.

Reply:
I have updated the e2e test.